core/src/oauth/connection.rs

```diff
@@ -1,5 +1,6 @@
 use crate::oauth::providers::{
     confluence::ConfluenceConnectionProvider, google_drive::GoogleDriveConnectionProvider,
```

Comment:
meed to merge with the one below :)

Reply:
WDYM ?

Reply:
Imports should be grouped

Reply:
In what way are they not groupped right now?

Reply:
Nvm me i didnt see the github one below